src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ProviderChooser.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {FC, useContext} from 'react'
+import {
+  AlignItems,
+  ComponentSize,
+  FlexBox,
+  FlexDirection,
+  InputLabel,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+// Style
+import './ProviderChooser.scss'
+
+// Components
+import {CreateOrgContext} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationContext'
+import {ClusterBox} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/ClusterBox'
+import {RegionDropdown} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/RegionDropdown'
+
+export const ProviderChooser: FC = () => {
+  const {clusters} = useContext(CreateOrgContext)
+  const clusterKeys = Object.keys(clusters)
+
+  return (
+    <FlexBox
+      alignItems={AlignItems.FlexStart}
+      justifyContent={JustifyContent.Center}
+      direction={FlexDirection.Column}
+      margin={ComponentSize.Large}
+      stretchToFitWidth={true}
+      className="provider-and-region-chooser"
+    >
```

Comment:
Super minor, but would move `className` up in the prop list.

Reply:
absolutely